Code/server/Services/InventoryService.cpp

```diff
@@ -36,7 +41,11 @@ void InventoryService::OnInventoryChanges(const PacketEvent<RequestInventoryChan
     NotifyInventoryChanges notify;
     notify.ServerId = message.ServerId;
     notify.Item = message.Item;
-    notify.Drop = message.Drop;
+
+    if (bEnableItemDrops)
```

Comment:
These 4 lines could just be

notify.Drop = bEnableItemDrops ? message.Drop : false;